File: examples/twitter/timeline/go/getATwitterTimeline.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/twitter"
)

// Get the timeline for a given user
func GetAtwitterTimeline() {
	twitterService := twitter.NewTwitterService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := twitterService.Timeline(&twitter.TimelineRequest{
		Limit:    1,
		Username: "m3oservices",
	})
	fmt.Println(rsp, err)
}

File: examples/twitter/timeline/node/getATwitterTimeline.js
const { TwitterService } = require("m3o/twitter");

// Get the timeline for a given user
async function getAtwitterTimeline() {
  let twitterService = new TwitterService(process.env.MICRO_API_TOKEN);
  let rsp = await twitterService.timeline({
    limit: 1,
    username: "m3oservices",
  });
  console.log(rsp);
}

getAtwitterTimeline();
